.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - development
```

Comment:
We don't actually have a dev lambda S3 bucket, so I think we should remove this.